src/input.cpp

```diff
@@ -1355,6 +1356,40 @@ cata::optional<tripoint> input_context::get_coordinates( const catacurses::windo
 }
 #endif
 
+std::tuple<point, bool> input_context::get_coordinates_text( const catacurses::window
+        &capture_win ) const
+{
+#if !defined( TILES )
+    static_cast<void>( capture_win );
```

Comment:
For consistency with most other places:

```suggestion
    ( void ) capture_win;
```